core/vm/actor/builtin/types/universal/universal.hpp

```diff
@@ -25,9 +25,30 @@ namespace fc::vm::actor::builtin::types {
   struct Universal : vm::actor::WithActorVersion {
     Universal() = default;
     explicit Universal(ActorVersion v) : WithActorVersion{v}, object{make(v)} {}
+
     Universal(ActorVersion v, std::shared_ptr<T> obj)
         : WithActorVersion{v}, object{obj} {}
 
+    Universal(const Universal<T> &other)
+        : WithActorVersion{other.actor_version},
+          object{other.object} {}
+
+    Universal(Universal<T> &&other) noexcept
+        : WithActorVersion{other.actor_version},
+          object{std::move(other.object)} {}
+
+    Universal<T> &operator=(const Universal<T> &other) {
+      actor_version = other.actor_version;
+      object = other.object;
+      return *this;
+    }
+
+    Universal<T> &operator=(Universal<T> &&other) noexcept {
+      actor_version = other.actor_version;
+      object = std::move(other.object);
+      return *this;
+    }
+
```

Comment:
Are implicit constructor/operator not working?

Reply:
There was an issue with cbor encoding, but it was fixed in cbor.